coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -183,9 +183,37 @@
 )
 
 
+class BuildInfo(object):
+    def __init__(self, id, buildrootid):
+        self.buildrootid = buildrootid
+        self.id          = id
```

Comment:
`id` is a builtin function as well, we should use something else in this function (leaving the class variable named `id` is fine though), e.x.:

```
def __init__(self, id_, buildrootid):
  self.id = id_
```

Reply:
fixed in next upload